### PR TITLE
#2.3 water

### PR DESCRIPTION
**Pull request by Manfred-Hastmark**
Added water to the planets, note that the sun also has water, this shouldn't be a problem as the sun will not be a planet rather a sphere in the future. Also the waves are only done with normalmaps and the actual mesh is not altered. This is because this would require a mesh with higher resolution to create this, which would require it to be built up into chunks or something to be performant and the task was only basic water.

image

### CODE REVIEW

File: Assets/Assets/PlanetTerrain/TEST.asset [hunk 1]
```diff
@@ -22,7 +22,7 @@ RenderTexture:
   m_ColorFormat: 49
   m_MipMap: 0
   m_GenerateMips: 1
-  m_SRGB: 1
```
Comment (Elin1043) [inline Assets/Assets/PlanetTerrain/TEST.asset:25]: Is this asset still used? Maybe you could remove it if you are not using it:)

Comment (Elin1043) [reply Assets/Assets/PlanetTerrain/TEST.asset:25]: Otherwise I think it looks good. So I can merge it after you answer this little thing.

Comment (Manfred-Hastmark) [reply Assets/Assets/PlanetTerrain/TEST.asset:25]: No, they are indeed old, I have removed them now and pushed up the branch

Comment (Elin1043) [reply Assets/Assets/PlanetTerrain/TEST.asset:25]: Nice! Good job!

**Pull request merged**